src/services/pagoPAProxyService.ts

```diff
@@ -59,7 +59,9 @@ export default class PagoPAProxyService {
     try {
       const client = this.pagoPAClient.getClient();
 
-      const res = await client.activatePayment({ payload });
+      const res = await client.activatePayment({
```

Comment:
this was simply wrong (look at the api definition) so the whole activatePayment procedure cannot work as is.

Reply:
as a side note, this currently passes the type checking as the getClient() return type is actually inferred as 'unknow' : )